rpcs3/Emu/RSX/GL/GLGSRender.h

```diff
@@ -18,9 +18,11 @@ struct work_item
 	std::mutex guard_mutex;
 	
 	u32  address_to_flush = 0;
-	bool processed = false;
-	bool result = false;
-	bool received = false;
+	gl::texture_cache::cached_rtt_section *section_to_flush = nullptr;
+
+	volatile bool processed = false;
```

Comment:
Ah, ok. If there won't be an issue with cache coherence and it's just for compiler optimizations it's fine.